flake.nix

```diff
@@ -6,7 +6,7 @@
     nixpkgs-pinned.url = "github:nixos/nixpkgs/5deee6281831847857720668867729617629ef1f";
     flake-utils.url = "github:numtide/flake-utils";
     pnpm2nix = {
-      url = "github:nzbr/pnpm2nix-nzbr";
```

Comment:
I had to [pin the version of `remarshal`](https://github.com/ThomasK33/pnpm2nix-nzbr/commit/c1f0ceeb759af20c5c232a9414036fda29a28a53) as the 0.20 version does not support yaml 1.1 anymore, and would fail our frontend build.

Opening a PR to upstream soon.